blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "billing_account_id" {
+  description = "Billing account id."
+  type        = string
+}
+
+variable "bucket_name" {
+  description = "Create GCS Bucket."
+  type        = string
+  default     = null
+}
+
+variable "dataset_name" {
+  description = "Create BigQuery Datasets."
+  type        = string
+  default     = null
+}
+
+variable "env" {
+  description = "Environment (dev,stg,prd)."
+  type        = string
+  default     = "dev"
+}
```

Comment:
Why we need to have a dedicated variable here? we can you the prefix one and suggest the user to put the env as part of the prefix.